pkg/exporter/monitor.go

```diff
@@ -134,7 +135,7 @@ func (m monitor) GetMemberStatus(id driver.ServerID, member driver.ServerHealth)
 		return result, err
 	}
 
-	req.URL, err = setPath(member.Endpoint, shared.ArangoExporterStatusEndpoint)
```

Comment:
Change Please also `logger.Err(err).Error("GetMemberStatus error")` to Info level

Reply:
Done. Also dropped one unused file related to endpoints handling